pkg/reconciler/revision/resources/deploy.go

```diff
@@ -70,6 +70,12 @@ var (
 		},
 	}
 
+	certVolumeMount = corev1.VolumeMount{
+		MountPath: queue.CertDirectory,
+		Name:      "server-certs",
+		ReadOnly:  true,
```

Comment:
I guess users might want to protect this from application code.
Stealing the private key can cause a hole.